src/components/ExpensiPicker.js

```diff
@@ -27,27 +35,37 @@ class ExpensiPicker extends PureComponent {
 
     render() {
         const {
-            label, isDisabled, ...pickerProps
+            errorText,
+            hasError,
+            label,
+            isDisabled,
+            ...pickerProps
         } = this.props;
         return (
-            <View
-                style={[
-                    styles.expensiPickerContainer,
-                    this.state.isOpen && styles.borderColorFocus,
-                    isDisabled && styles.inputDisabled,
-                ]}
-            >
-                {label && (
-                    <Text style={[styles.expensiPickerLabel, styles.textLabelSupporting]}>{label}</Text>
+            <>
+                <View
+                    style={[
+                        styles.expensiPickerContainer,
+                        this.state.isOpen && styles.borderColorFocus,
+                        isDisabled && styles.inputDisabled,
+                        (errorText || hasError) && styles.borderColorDanger,
+                    ]}
+                >
+                    {label && (
+                        <Text style={[styles.expensiPickerLabel, styles.textLabelSupporting]}>{label}</Text>
+                    )}
+                    <Picker
+                        onOpen={() => this.setState({isOpen: true})}
+                        onClose={() => this.setState({isOpen: false})}
+                        disabled={isDisabled}
+                        // eslint-disable-next-line react/jsx-props-no-spreading
+                        {...pickerProps}
+                    />
+                </View>
+                {Boolean(errorText) && (
```

Comment:
Rather than use a Boolean cast can we check if the string (which defaults to `''`) is empty?

```js
!_.isEmpty(errorText) ? <Text /> : null
``` 

I think we are trying to standardize on this as per the advice [here](https://github.com/Expensify/App/blob/main/STYLE.md#important-note)

Reply:
Sounds good, I'll try to stick to that!

Reply:
>I feel like it is more boilerplate for the same, not making it more readable.

When in doubt I think we should just always error on the side of consistency. If we want to change how it's done then let's change the example in the style guide then update it everywhere.